Sources/PackageLoading/README.md

```diff
@@ -4,7 +4,7 @@ This library defines the logic which translates between the Swift package
 manager conventions and the underlying project model.
 
 The intent is that it is largely a transformation taking the input project model
-objects descripted in a manifest applying the conventions.
+objects described in a manifest.
```

Comment:
Not sure what "the conventions` meant in this context. This change is actually a follow up to my old PR https://github.com/apple/swift-package-manager/pull/2977, where I was trying to fix wording in this paragraph too.